src/Altinn.Common/Altinn.Common.PEP/Altinn.Common.PEP/Configuration/GeneralSettings.cs

```diff
@@ -0,0 +1,203 @@
+using System;
+
+namespace Altinn.Common.PEP.Configuration
+{
+    /// <summary>
+    /// General configuration settings
+    /// </summary>
+    public class GeneralSettings
+    {
+        /// <summary>
+        /// Gets or sets the location to search for templates
+        /// </summary>
+        public string TemplateLocation { get; set; }
```

Comment:
Do you really need all these values? I would suggest stripping it to the bare minimum.

Reply:
DisablePEP seems to be the only value you are using here